packages/block-library/src/post-terms/edit.js

```diff
@@ -92,7 +104,15 @@ export default function PostTermsEdit( {
 								{ postTerm.name }
 							</a>
 						) )
-						.reduce( ( prev, curr ) => [ prev, ' | ', curr ] ) }
+						.reduce( ( prev, curr ) => (
+							<>
+								{ prev }
+								<span className="wp-block-post-terms__sep">
```

Comment:
Also nitpick but let's name this with the whole word `separator`.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/32812/commits/36259d1406074f6cdb28129d06a9684d0ddc7a31 :+1: